### DISCUSSION TITLE
SSR and Module Federation example

### DISCUSSION THREAD

**Discussion by andrewmota** [web-infra-dev/rsbuild#2397] [Q&A] [answered]
Hi all,

Has anyone managed to create an rsbuild application with SSR and Module Federation enabled?

I was looking at some repositories from other discussions, like https://github.com/9aoy/rsbuild-ssr-example/tree/main and trying to enable something and I couldn't.

If anyone has any examples that can share, it would be very helpful.

**Comment by zhoushaw** [comment, marked as answer]
@andrewmota We are planning to provide the MF SSR solution based on rsbuild, which will be supported on the modern.js framework first, which we plan to provide RFC in the next few days

**Comment by jkzing** [comment]
We also have a Rsbuild + Module Federation example [here](https://github.com/rspack-contrib/rspack-examples/tree/main/rsbuild/module-federation-enhanced) (without SSR though), hope it helps.

**Comment by 2heal1** [comment]
@andrewmota Here is the [RFC](https://github.com/module-federation/core/issues/2348) , I'll probably finish it by 6.15 if everything goes well :D

**Comment by andrewmota** [comment]
Thank you all! I appreciate this.
